doc/explanation/containers_and_vms.md

```diff
@@ -3,22 +3,37 @@
 
 Incus provides support for two different types of {ref}`instances <expl-instances>`: *system containers* and *virtual machines*.
 
-When running a system container, Incus simulates a virtual version of a full operating system. To do this, it uses the functionality provided by the kernel running on the host system.
+When running a system container, Incus simulates a virtual version of a full operating system.  
+To do this, it uses the functionality provided by the kernel running on the host system.
 
-When running a virtual machine, Incus uses the hardware of the host system, but the kernel is provided by the virtual machine. Therefore, virtual machines can be used to run, for example, a different operating system.
+When running a virtual machine, Incus uses the hardware of the host system, and the kernel is provided by the virtual machine.  
+Therefore, virtual machines can be used to run, for example, a different operating system.
+ 
+|    Virtual Machines              |  Application Containers          |     System Containers    |      
+|      :----:                     |      :----:                     |        :----:           |
+| Uses a dedicated kernel                 |  Uses the kernel of the host            |   Uses the kernel of the host   |    
+| Can host different types of OS  |  Can only host Linux            |  Can only host Linux    |   
+| Uses more ressources            |  Uses less ressources           |  Uses less resources    |   
+| Can host multiple applications         |  Can host a single app |  Can host multiple applications |  
```

Comment:
Probably should be `Can host a single application` for consistency with the other fields